src/proton/layers/core/core.cljs

```diff
@@ -1,19 +1,22 @@
 (ns proton.layers.core.core
   (:use [proton.layers.base :only [init-layer! get-initial-config get-keybindings get-packages get-keymaps]])
-  (:require [proton.lib.proton :as proton]))
+  (:require [proton.lib.proton :as proton]
+            [proton.lib.package_manager :as package]))
 
 (def state (atom {}))
 
 (defn get-active-pane [atom]
   (.getView (.-views atom) (.getActivePane (.-workspace atom))))
 
-(defn toggle-visibility [element]
-  (if (nil? (.getAttribute element "style"))
-    (.setAttribute element "style" "display:none")
-    (.removeAttribute element "style")))
+(defn toggle-tabs [visible]
```

Comment:
Just thinking about the future...

Would it make sense to define a `toggle-package` function that contains the body of these `toggle-PACKAGE-NAME` functions?